web/src/components/Incidents/utils.ts

```diff
@@ -168,22 +230,24 @@ export const createIncidentsChartBars = (incident, theme, dateArray) => {
   return data;
 };
 
-function consolidateAndMergeAlertIntervals(data, dateArray) {
-  const sortedValues = data.values.sort((a, b) => new Date(a[0]) - new Date(b[0]));
+function consolidateAndMergeAlertIntervals(data: Alert, dateArray: SpanDates) {
+  const sortedValues = data.values.sort(
+    (a, b) => new Date(a[0]).valueOf() - new Date(b[0]).valueOf(),
+  );
 
-  const intervals = [];
+  const intervals: Array<AlertsIntervalsArray> = [];
   let currentStart = sortedValues[0][0],
     previousTimestamp = new Date(currentStart);
 
   for (let i = 1; i < sortedValues.length; i++) {
     const currentTimestamp = new Date(sortedValues[i][0]);
-    const timeDifference = (currentTimestamp - previousTimestamp) / 60000; // Convert to minutes
+    const timeDifference = (currentTimestamp.valueOf() - previousTimestamp.valueOf()) / 60000; // Convert to minutes
```

Comment:
We should probably use one of `valueOf()` or `getTime()` but not both since the function returns the same value. I'm in favor of sticking with `getTime()` as is more clear what the intention is.

Reply:
Thanks I will update the code